### PR TITLE
update  Apache_servicecomb HuaweiConnect post 

### PR DESCRIPTION
**Pull request by imlidian**

### CODE REVIEW

**Comment by imlidian** [comment]
you can review with this link. https://imlidian.github.io/cn/docs/Apache-ServiceComb-Meetup-2018-Shanghai-Report/

File: _posts/cn/2018-09-28-Apache-ServiceComb-Meetup-2018-Shanghai.md [hunk 1]
```diff
@@ -34,7 +34,8 @@ ServiceComb Apache软件基金会孵化项目的主体代码由华为云捐赠
 ---
 **《Apache之道》**    
 
-**时间：** 13:30-14:30
+[中文PDF下载](/assets/slides/20181012/1_Apache_HuaweiConnect_Presentation_20181012.pdf)  [英文PDF下载](/assets/slides/20181012/EN/1_Apache_HuaweiConnect_Presentation_20181012.pdf)
```
Comment (WillemJiang) [inline _posts/cn/2018-09-28-Apache-ServiceComb-Meetup-2018-Shanghai.md]: 英文说明上没有添加下载链接。 建议英文版添加英文下载链接，中文版添加中文下载链接。

File: _pages/cn/home.md [hunk 2]
```diff
@@ -38,7 +38,7 @@ intro:
     <a href='/cn/docs/quick-start-dataconsistency/'>号外：现在ServiceComb提供了微服务场景下的数据一致性解决方案Saga！</a>
   </li>
   <li>
-    <a href='/cn/docs/Apache-ServiceComb-Meetup-2018-Shanghai/'>Apache ServiceComb Meetup -上海 2018.10.12</a>
+    <a href='/cn/docs/Apache-ServiceComb-Meetup-2018-Shanghai-Report/'>Apache ServiceComb Meetup -上海 2018.10.12 Report(ppt)</a>
```
Comment (zenlinTechnofreak) [inline _pages/cn/home.md]: Can you write clearly what is meant by the brackets?
是否可以写清楚括号内是什么？比如，（内含演讲稿下载链接）

Comment (zenlinTechnofreak) [reply _pages/cn/home.md]: Please do not use Chinese in the code.
代码里面避免出现中文字眼。

File: _pages/home.md [hunk 3]
```diff
@@ -35,7 +35,7 @@ intro:
     <a href='/release/saga-downloads/'>Saga releases a new version 0.2.0</a>
   </li>
   <li>
-    <a href='/docs/Apache-ServiceComb-Meetup-2018-Shanghai/'>Apache ServiceComb Meetup -Shanghai 2018.10.12</a>
+    <a href='/docs/Apache-ServiceComb-Meetup-2018-Shanghai-Report/'>Apache ServiceComb Meetup -Shanghai 2018.10.12 Report(PPT)</a>
```
Comment (zenlinTechnofreak) [inline _pages/home.md]: Please clearfy what '(ppt)' means?

File: _posts/cn/2018-09-28-Apache-ServiceComb-Meetup-2018-Shanghai.md [hunk 4]
```diff
@@ -34,7 +34,8 @@ ServiceComb Apache软件基金会孵化项目的主体代码由华为云捐赠
 ---
 **《Apache之道》**
```
Comment (zenlinTechnofreak) [inline _posts/cn/2018-09-28-Apache-ServiceComb-Meetup-2018-Shanghai.md]: Could you create a new blog for activity summary and  ppt download, but not just modify the blog for upcoming event?
现在写的是一篇新的活动总结和PPT下载的博客，不是在互动预告的博客上编辑。请帮忙继续处理一下。


I find that there are a new blog for activity summary and ppt download, please do not change blog, we have no need to change this blog.
看到后面发现有新的博客了，这个活动预告的博客不要更改了。请去除这整个老的blog的更改。

File: _posts/cn/2018-10-29-Apache-ServiceComb-Meetup-2018-Shanghai-Report.md [hunk 5]
```diff
@@ -0,0 +1,80 @@
...
+author: Dean Lee
+tags: [microservice, Meetup]
+redirect_from:
+  - /theme-setup/
+---
+![1](/assets/images/hc2018/1.jpeg)
+
+**《Apache之道》**    
+
+[PDF下载](/assets/slides/20181012/EN/1_Apache_HuaweiConnect_Presentation_20181012.pdf)
```
Comment (zenlinTechnofreak) [inline _posts/cn/2018-10-29-Apache-ServiceComb-Meetup-2018-Shanghai-Report.md]: Could you change the link name of "1_Apache_HuaweiConnect_Presentation_20181012.pdf" to the correct one？
You can just use the topic name for it.

File: _posts/cn/2018-10-29-Apache-ServiceComb-Meetup-2018-Shanghai-Report.md [hunk 6]
```diff
@@ -0,0 +1,80 @@
...
+excerpt: "Apache ServiceComb Meetup -Shanghai 2018.10.12 Report (PPT)"
+last_modified_at: 2018-10-29T17:18:43+08:00
+author: Dean Lee
+tags: [microservice, Meetup]
+redirect_from:
+  - /theme-setup/
+---
+![1](/assets/images/hc2018/1.jpeg)
+
+**《Apache之道》**
```
Comment (zenlinTechnofreak) [inline _posts/cn/2018-10-29-Apache-ServiceComb-Meetup-2018-Shanghai-Report.md]: Topic name and introduction should be before the picture.

File: _posts/cn/2018-10-29-Apache-ServiceComb-Meetup-2018-Shanghai-Report.md [hunk 7]
```diff
@@ -0,0 +1,80 @@
...
+permalink: /cn/docs/Apache-ServiceComb-Meetup-2018-Shanghai-Report/
+
+excerpt: "Apache ServiceComb Meetup -Shanghai 2018.10.12 Report (PPT)"
+last_modified_at: 2018-10-29T17:18:43+08:00
+author: Dean Lee
+tags: [microservice, Meetup]
+redirect_from:
+  - /theme-setup/
+---
+![1](/assets/images/hc2018/1.jpeg)
```
Comment (zenlinTechnofreak) [inline _posts/cn/2018-10-29-Apache-ServiceComb-Meetup-2018-Shanghai-Report.md]: Could you have a Conclusion for the meetup?such as,  what have been introduced in the meetup, how many people joined the meetup on site and on the wbcast ,  how about the atmosphere and the participation in the scense?

File: _posts/cn/2018-10-29-Apache-ServiceComb-Meetup-2018-Shanghai-Report.md [hunk 8]
```diff
@@ -0,0 +1,80 @@
...
+
+**《快速微服务化编程体验》**
+
+[中文PDF下载](/assets/slides/20181012/6_Experience_on_Fast_Microservice_Programming.pdf)  [英文PDF下载](/assets/slides/20181012/EN/6_Experience_on_Fast_Microservice_Programming.pdf)
+
+**巨震：** 现场演示如何快速使用ServiceComb Java-chassis和Service Mesh技术Mesher进行微服务化应用编程，结合Kubernetes进行快速部署，并和与会者互动练习。
+
+---
+
+关于Apache ServiceComb 项目**：
```
Comment (zenlinTechnofreak) [inline _posts/cn/2018-10-29-Apache-ServiceComb-Meetup-2018-Shanghai-Report.md]: .MD syntax error here.

File: _posts/cn/2018-09-28-Apache-ServiceComb-Meetup-2018-Shanghai.md (hunk 1 above)
Comment (imlidian) [reply _posts/cn/2018-09-28-Apache-ServiceComb-Meetup-2018-Shanghai.md]: 没有必要，用户想下载两个版本还需要切换连接，不方便

File: _pages/cn/home.md (hunk 2 above)
Comment (imlidian) [reply _pages/cn/home.md]: 1 (ppt download) (ppt inside) 或者 (ppt link) 给个措辞吧。我看国外就一个ppt大家就都明白了。
2 那不是代码，就是link的content

File: _pages/home.md (hunk 3 above)
Comment (imlidian) [reply _pages/home.md]: same as before

File: _posts/cn/2018-09-28-Apache-ServiceComb-Meetup-2018-Shanghai.md (hunk 1 above)
Comment (zenlinTechnofreak) [reply _posts/cn/2018-09-28-Apache-ServiceComb-Meetup-2018-Shanghai.md]: +1 to WillemJiang's suggestion.

File: _pages/cn/home.md (hunk 2 above)
Comment (zenlinTechnofreak) [reply _pages/cn/home.md]: 请处理。link content 为什么用中文？

Comment (zenlinTechnofreak) [reply _pages/cn/home.md]: ppt download 可以

File: _posts/cn/2018-10-29-Apache-ServiceComb-Meetup-2018-Shanghai-Report.md (hunk 6 above)
Comment (imlidian) [reply _posts/cn/2018-10-29-Apache-ServiceComb-Meetup-2018-Shanghai-Report.md]: 之前试了，看着不太顺。这个不改了

Comment (imlidian) [reply _posts/cn/2018-10-29-Apache-ServiceComb-Meetup-2018-Shanghai-Report.md]: done

File: _posts/cn/2018-10-29-Apache-ServiceComb-Meetup-2018-Shanghai-Report.md (hunk 7 above)
Comment (imlidian) [reply _posts/cn/2018-10-29-Apache-ServiceComb-Meetup-2018-Shanghai-Report.md]: done

**Comment by imlidian** [comment]
you can review with this link. https://imlidian.github.io/

**Pull request merged**